openfermioncirq/gates/four_qubit_gates_test.py

```diff
@@ -124,7 +176,55 @@ def test_double_excitation_decompose(half_turns):
                       3 / numpy.sqrt(2) - 1j * numpy.sqrt(2), 0, 0, 1]) /
          numpy.sqrt(15),
          5e-6)
-    ])
+    ]
+combined_double_excitation_simulator_test_cases = [
+        (CombinedDoubleExcitationGate((0, 0, 0)), 1.,
+         numpy.ones(16) / 4.,
+         numpy.ones(16) / 4.,
+         5e-6),
+        (CombinedDoubleExcitationGate((0.2, -0.1, 0.7)), 0.,
```

Comment:
Maybe just add one final test identical to this, but with `half_turns=0.3` or something, that should totally cover everything for the simulator!

Reply:
Done